index.js

```diff
@@ -48,8 +48,7 @@ function newProcessor(context, opConfig, jobConfig) {
             }, {concurrency: 1})
             .catch(function(err) {
                 //for now we will throw if there is an async error
-                var errMsg = err.stack ? err.stack : err
-                return Promise.reject(`Error sending data to file: ${filename}, error: ${errMsg}, data: ${JSON.stringify(chunks)}`)
+                return Promise.reject(`Error sending data to file: ${filename}, error: ${err.message}, stack: ${err.stack}, data: ${JSON.stringify(chunks)}`)
```

Comment:
The deleted line, 51, leads me to believe there may be cases where `err.stack` doesn't exist, if that's true, then won't this line result in an undefined error?

Reply:
It will resolve as `undefined` without error:
```javascript
> e = new Error('OHAI!')
Error: OHAI!
    at repl:1:5
    at sigintHandlersWrap (vm.js:22:35)
    at sigintHandlersWrap (vm.js:73:12)
    at ContextifyScript.Script.runInThisContext (vm.js:21:12)
    at REPLServer.defaultEval (repl.js:340:29)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.<anonymous> (repl.js:538:10)
    at emitOne (events.js:101:20)
    at REPLServer.emit (events.js:188:7)
> delete e.stack
true
> `Should resolve to null: ${e.stack}`
'Should resolve to null: undefined'
```

Reply:
Apparently there is a new common way that we handle errors like this in Teraslice code, it goes something like this:

```
const errMsg = parseError(err);
logger.error('could not create execution context', errMsg);
return Promise.reject(errMsg);
```

It's in this package here:

https://github.com/terascope/error_parser/blob/master/index.js

If that satisfies your needs try and use that, if not, we can go with this PR as is.

Reply:
Doh!  Just now realizing that Error.stack does include the message.  The lack of that in some of the stacks we're seeing led me down this path.  This PR adds no value.